dev-docs/plugins/cross-player-prebid-component/cross-player-config.md

```diff
@@ -0,0 +1,411 @@
+---
+layout: page_v2
+title: Cross-Player Prebid Component Configuration
+description: Configuring the cross-player Prebid component
+top_nav_section: dev_docs
+nav_section: plugins
+---
+
+# Cross Player Prebid Component Configuration
+{:.no_toc}
+
+Options that configure the behavior of the Prebid process are passed to the [Cross-Player Prebid Component]({{site.baseurl}}/dev-docs/plugins/cross-player-prebid-component/about-cross-player-prebid-component.html) in a JSON structure.  These options include the bidders and bidder settings for Prebid.js to use, the URL to your custom Prebid.js build, and any ad server parameters.
+
+The Prebid Component supports almost all the Prebid options supported by the other Prebid plugins, with some exceptions.  The following is the list of Prebid options supported in the Component:
+
+- [prebidPath](#prebidPath)
+- [biddersSpec](#biddersSpec)
+- [bidderAliases](#bidderAliases)
+- [bidderSettings](#bidderSettings)
+- [prebidConfigOptions](#prebidConfigOptions)
+- [dfpParameters](#dfpParameters)
```

Comment:
This should be gptParameters. We shouldn't be using 'DFP' anymore.

Reply:
This parameter was named dfpParameters when we built the original Brightcove Player prebid plugin because these parameters were used when we called the following method on prebid.js:  pbjs.adServers.dfp.buildVideoUrl(options).  See http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.adServers.dfp.buildVideoUrl  That method has not changed its name so we have not changed the name of the parameter.  We can either change the name of our parameter - which would require we change the code in the "plugin" or we can have two entry points which do the same thing (UGLY).   Perhaps it would help if the doc about this parameter should simply say something about GAM being previously known as DFP.

It should also be noted that the older Brightcove prebid plugin still uses the dfpParameters parameter - this is a product that is currently in use.

Reply:
About the "term" plugin - you are correct - this is technically NOT a plugin.  Indeed we have been calling it a Cross Player Prebid Component.  So we should probably replace the use of the term "plugin" with component.

This component was built to help a specific customer get started with a significant prebid video integration.  They had asked for some help and some guidance and this component was written to give them a model for achieving their goals.

Reply:
I have modified the internal source documents - replacing the word plugin with the word component where possible.  

However, the word plugin does appear in the url path and the name of the component.  I am unable to change those names at this time because I do not want to affect the testing that the customer is doing.

These changes may be made in the doc that is present in this pull request.

Reply:
Cool - so who's going to make the changes here -- @jeanstemp ?